README.md

```diff
@@ -46,18 +76,38 @@ cmake ../
 make -j
 ```
 
-After that, in the `build` folder will be the compiled library code as well
-as generated samples.
+On Windows there are a couple of options worth looking into. CMake has a GUI
+which allows you to select projects and how to build them, if you use the GUI,
+you can directly select "MinGW" as the generator to ensure that the build
+takes place properly. Or you can make use of CLion which will abstract the
+build process away. Another option (which I recommend for long term use), is
+to learn about WSL (Windows Subsystem for Linux). Linux is the OS of choice
+for software developers and includes a lot of great tools for development. WSL
+is a happy medium between running Windows and installing a virtual machine.
 
-For building with linting enabled (recommended). First install cpplint, then
-run the installation with the modification below.
+After running those commands, you should have a fully build EVT-core including
+samples.
 
-```bash
-mkdir build/
-cd build/
-cmake -DEVT_LINT=ON ../
-make -j
-```
+For a more detailed build process using CLion and on Windows,
+visit the EVT wiki.
+
+#### Step 4. Flashing the Board
+
+After building has taken place, you should be left with a folder that contains
+all of the built samples. The samples will be located in `build/samples` and
+each sample with have its own folder. 
+
+For example, in `build/samples/blink/` there will be an ELF, bin, hex, and
+map files among others. To flash the blink sample to the nucleo, copy the
+bin file from your directory to the nucleo that is connected to their computer.
+The nucleo should show up as an external storage device. After dragging the
+bin file to the nucleo, the sample should be flashed and an LED on the board
+should start blinking at a set rate.
+
+For interacting with samples that make use of a UART monitor, a serial console
+will be needed. PuTTY on Windows is a good tool. Linux and MacOS will have some
+built in tools for serial console. But when in doubt, the Arduino serial
```

Comment:
After some testing, I have really come to enjoy gtkterm on Linux.  It could be worth suggesting, but I'm fine with leaving it open ended.

Reply:
Good to know, I'll add it in!